docs/setup.asciidoc

```diff
@@ -1,26 +1,49 @@
 [[setup]]
-== Getting Kibana Up and Running
+== Installing Kibana
 You can set up Kibana and start exploring your Elasticsearch indices in minutes.
 All you need is:
 
 * Elasticsearch {esversion}
-* A modern web browser - http://www.elastic.co/subscriptions/matrix#matrix_browsers[Supported Browsers].
+* A modern web browser - https://www.elastic.co/support/matrix#show_browsers[Supported Browsers].
 * Information about your Elasticsearch installation:
 ** URL of the Elasticsearch instance you want to connect to.
 ** Which Elasticsearch indices you want to search.
 
 NOTE: If your Elasticsearch installation is protected by http://www.elastic.co/overview/shield/[{scyld}], see
 {shield}/kibana.html#using-kibana4-with-shield[{scyld} with Kibana] for additional setup instructions.
 
+=== Upgrading Kibana
+
+Your existing Kibana version is generally compatible with the next minor version release of Elasticsearch.
+This means you should upgrade your Elasticsearch cluster(s) before or at the same time as Kibana.
+We cannot guarantee compatibility between major version releases so in those cases both Elasticsearch and Kibana must be upgraded together.
+
+. Create a https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-snapshots.html[snapshot] of the existing .kibana index.
+. Back up the `kibana.yml` configuration file.
+. Take note of the Kibana plugins that are installed;
+ * `bin/kibana plugin --list` on 4.x versions of Kibana.
+ * `bin/kibana-plugin list` on 5.x versions of Kibana.
+. To upgrade from an Archive File:
+ * Extract the new version of Kibana into a different directory.  See steps below.
```

Comment:
I'd either remove "See steps below" or make it a real cross-reference. (I don't think it's really needed, I'd just take it out.) 

Also, I'd put the upgrade instructions after the install instructions.